bin/strata-reth/src/main.rs

```diff
@@ -107,6 +106,20 @@ pub struct AdditionalConfig {
     pub sequencer_http: Option<String>,
 }
 
+#[derive(Debug, Clone, Default)]
+#[non_exhaustive]
+pub struct StrataChainSpecParser;
+
+impl ChainSpecParser for StrataChainSpecParser {
+    type ChainSpec = ChainSpec;
+
+    const SUPPORTED_CHAINS: &'static [&'static str] = &["dev", "devnet"];
+
+    fn parse(s: &str) -> eyre::Result<Arc<Self::ChainSpec>> {
+        chain_value_parser(s)
+    }
+}
```

Comment:
I think this should be an `enum` with the `#[non_exhaustive]` and the `SUPPORTED_CHAINS` will be the variants.
This makes it easier to catch in `semver-checks` in the future.